x/nft/types/nft.go

```diff
@@ -32,9 +33,8 @@ func (bnft BaseNFT) GetName() string {
 }
 
 // GetOwner return the owner of BaseNFT
-func (bnft BaseNFT) GetOwner() sdk.AccAddress {
-	owner, _ := sdk.AccAddressFromBech32(bnft.Owner)
-	return owner
+func (bnft BaseNFT) GetOwner() (sdk.AccAddress, error) {
```

Comment:
does the BaseNFT.owner need to be "string" -- could it be `sdk.AccAddress` instead?

Reply:
I guess it's from protobuf, so needs string?

Reply:
Yes. This is from protobuf so it needs to be a string.